doc/users/prev_whats_new/whats_new_3.3.0.rst

```diff
@@ -185,83 +153,210 @@ for out-of-range values with colors that differ from adjacent in-range colors.
+---------------------------------------------------------------------------
+
+`~.axes.Axes.set_title` tries to auto-position the title to avoid any
+decorators on the top x-axis. This is not always desirable so now *y* is an
+explicit keyword argument of `~.axes.Axes.set_title`. It defaults to *None*
+which means to use auto-positioning. If a value is supplied (i.e. the pre-3.0
+default was ``y=1.0``) then auto-positioning is turned off. This can also be
+set with the new rcParameter :rc:`axes.titley`.
+
+.. plot::
+
+    fig, axs = plt.subplots(1, 2, constrained_layout=True, figsize=(5, 2))
+    axs[0].set_title('y=0.7\n$\sum_{j_n} x_j$', y=0.7)
+    axs[1].set_title('y=None\n$\sum_{j_n} x_j$')
+    plt.show()
+
+
+Dates now use a modern epoch
+----------------------------
+
+Matplotlib converts dates to days since an epoch using `.dates.date2num` (via
+`matplotlib.units`). Previously, an epoch of ``0000-12-31T00:00:00`` was used
+so that ``0001-01-01`` was converted to 1.0. An epoch so distant in the
+past meant that a modern date was not able to preserve microseconds because
+2000 years times the 2^(-52) resolution of a 64-bit float gives 14
+microseconds.
+
+Here we change the default epoch to the more reasonable UNIX default of
+``1970-01-01T00:00:00`` which for a modern date has 0.35 microsecond
+resolution.  (Finer resolution is not possible because we rely on
+`datetime.datetime` for the date locators). Access to the epoch is provided
+by `~.dates.get_epoch`, and there is a new :rc:`date.epoch` rcParam. The user
+may also call `~.dates.set_epoch`, but it must be set *before* any date
+conversion or plotting is used.
+
+If you have data stored as ordinal floats in the old epoch, a simple
+conversion (using the new epoch) is::
+
+    new_ordinal = old_ordinal + mdates.date2num(np.datetime64('0000-12-31'))
+
+
+tight_layout now supports suptitle
+----------------------------------
+
+Previous versions did not consider `.Figure.suptitle`, and so it may overlap
+with other artists after calling `~.Figure.tight_layout`:
+
+.. plot::
+
+    fig, axs = plt.subplots(1, 3)
+    for i, ax in enumerate(axs):
+        ax.plot([1, 2, 3])
+        ax.set_title(f'Axes {i}')
+
+    t = fig.suptitle('suptitle')
+    t.set_in_layout(False)
+    fig.tight_layout()
+
+From now on, the ``suptitle`` will be considered:
+
+.. plot::
+
+    fig, axs = plt.subplots(1, 3)
+    for i, ax in enumerate(axs):
+        ax.plot([1, 2, 3])
+        ax.set_title(f'Axes {i}')
+
+    fig.suptitle('suptitle')
+    fig.tight_layout()
+
+
+Allow tick formatters to be set with str or function inputs
+-----------------------------------------------------------
+
+`~.Axis.set_major_formatter` and `~.Axis.set_minor_formatter`
+now accept `str` or function inputs in addition to `~.ticker.Formatter`
+instances. For a `str` a `~.ticker.StrMethodFormatter` is automatically
+generated and used. For a function a `~.ticker.FuncFormatter` is automatically
+generated and used.
+
```

Comment:
I think a mini code example might worth adding here, something like
```python
import matplotlib.ticker as mticker
mticker.Formatter("price: {:02}")
mticker.Formatter(lambda x, _ :  "low" if x<2 else "high")
```
also this should maybe go higher...

Reply:
sorry, gave you code that won't work. Here's the fix
````python
ax.xaxis.set_major_formatter("price: {x:02}")
ax.yaxis.set_minor_formatter(lambda x, pos :  "low" if x<2 else "high")
````

Reply:
@tacaswell made the suggestion that it might be worth (with hiding the code) to embed a modified version of the first plot in the updated [tick formatter example](https://matplotlib.org/devdocs/gallery/ticks_and_spines/tick-formatters.html#sphx-glr-gallery-ticks-and-spines-tick-formatters-py) with the label replaced with a longer version that includes the `ax.set_{major,minor}_formatter`. 
![image](https://user-images.githubusercontent.com/1300499/87249900-ccff1e80-c42f-11ea-924e-3449602c7e97.png)

Reply:
Added.